### PR TITLE
More explicit instruction around sources and layers.

### PR DESCRIPTION
**Pull request by tmcw**
cc @lucaswoj for the review

### CODE REVIEW

File: docs/_generate/index.html [hunk 1]
```diff
@@ -139,7 +139,15 @@
       <h2><a href='#sources' title='link to sources'>Sources</a></h2>
       <p>
         Sources supply data to be shown on the map. The type of source is specified by the <code>"type"</code> property,
-        and must be one of <var><%= source_types.join(', ') %></var>. Tiled sources (vector and raster) must specify
+        and must be one of <var><%= source_types.join(', ') %></var>. Sources
+        supply data but don't make it appear on the map - they don't contain
```
Comment (lucaswoj) [inline docs/_generate/index.html]: This sentence is a little repetitive to the first sentence

> Sources supply data to be shown on the map.
> 
> Sources supply data but don't make it appear on the map...

File: docs/_generate/index.html [hunk 2]
```diff
@@ -139,7 +139,15 @@
...
+        any styling details like color or width. Instead, you define Layers that refer
+        to a source and give it a visual representation. This makes it possible
```
Comment (lucaswoj) [inline docs/_generate/index.html:145]: No need to complicate this with a negation

> ~~Instead, you define~~ Layers ~~that~~ refer to a source and give it a visual representation.

**Comment by tmcw** [comment]
👍 resolved the comments

**Comment by lucaswoj** [comment]
Looks great! Thank you @tmcw! :shipit:

**Pull request merged**